Lib/http/__init__.py

```diff
@@ -114,6 +115,10 @@ def __new__(cls, value, phrase, description=''):
         'Request Header Fields Too Large',
         'The server is unwilling to process the request because its header '
         'fields are too large')
+    UNAVAILABLE_FOR_LEGAL_REASONS = (451,
```

Comment:
Please add this constant to list of tested constants like the previously added constant MISDIRECTED_REQUEST : https://github.com/python/cpython/pull/2589/files#diff-4334c6b7e8fcfc88f2555f65fa7ab337R1358

Reply:
Done.